chain/jsonrpc/res/last_blocks.html

```diff
@@ -87,8 +88,8 @@
                         for (let i = 0; i < (max_shard + 1) * 3; i += 1) {
                             shards_cells.push($('<td>'));
                         }
-                        element.chunks.forEach(chunk => {
-                            shards_cells[chunk.shard_id * 3].append(createHashElement(chunk.chunk_hash, chunk.chunk_producer));
+                        element.chunks.forEach((chunk, index) => {
+                            shards_cells[chunk.shard_id * 3].append(createHashElement(chunk.chunk_hash, chunk.chunk_producer, "chunk_" + index));
```

Comment:
Why not just have `let chunks_by_index = {};` outside of this for loop and then:

```javascript
let element = createHashElement(chunk.chunk_hash, chunk.chunk_producer)
(chunks_by_shard[index] = chunks_by_shard[index] || []).push(element);
shards_cells[chunk.shard_id * 3].append(element);
```

Using DOM seems unnecessary.

Reply:
done